includes/footer.html

```diff
@@ -60,7 +60,14 @@ <h5 class="white-text fine">coala</h5>
     </div>
     <div class="footer-copyright">
         <div class="container">
-            <span class="left">© 2017 coala</span>
+            <span class="left">
+	        ©
+		<script>
+		    document.getElementById("current-year").innerHTML=new Date().getFullYear();
+		</script>
+		<span id="current-year"></span>
```

Comment:
Indentation: Do not use tabs: Remove the tabs.

*Origin: HTMLLintBear, Section: `html`.*